lib/fileHandler/index.js

```diff
@@ -4,251 +4,255 @@ var util   = require('util');
 var when   = require('when');
 var guid   = require('mout/random/guid');
 var mime   = require('mime');
+var moment = require('moment');
 
 var needle = require('needle');
-var knox   = require('knox');
+
+var AWS = require('aws-sdk');
+
+var REGION = 'us-west-2',
+	BUCKET = 'workflow-file-uploads';
 
 
 module.exports = function (options) {
 
+	//Handling development mode
+	function checkDevConfig () {
+		if (process.env.NODE_ENV === 'development' && _.isUndefined(options.aws) && _.isUndefined(process.env.AWS_ACCESS_KEY_ID)) {
+			return new Error('For development and testing please require an aws.json file with `key` and `secret` or add `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to the environment.');
```

Comment:
There's no need, as this would only occur if the code is run with the development flag, i.e. on the local machine. The code on lambda only ever runs in production mode.

As for `AWS_PROFILE`, I can add a check to see if it exists.